modules/administration-guide/partials/proc_installing-image-puller-using-openshift-templates.adoc

```diff
@@ -40,7 +40,7 @@ include::example$snip_{project-context}-cloning-image-puller-project.adoc[]
 
 |`SERVICEACCOUNT_NAME` 
 |The name of the ServiceAccount created and used by the deployment
-|`{image-puller-serviceaccount-name}`
```

Comment:
Wouldn't it make more sense to define a proper value for the `image-puller-serviceaccount-name` attribute (and keep it). Is the value really based on the name of the repository?

Reply:
@boczkowska works with attributes lately. I am not sure what is leftover and what is just not finish work yet. Just did what makes sense.

Reply:
Based on the info from Ilya @ibuziuk there is no need to attribute this since the value is the same for Che and CRW.
https://github.com/che-incubator/kubernetes-image-puller/blob/master/deploy/openshift/serviceaccount.yaml#L6

Reply:
Fixing this in with one of the commits I preparade here